framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -314,6 +315,9 @@ public class AnnotatedTypeFactory implements AnnotationProvider {
      */
     protected @Nullable AnnotationFileElementTypes currentFileAjavaTypes;
 
+    /** Alias files provided via -Aaliases={files} */
+    private List<File> aliasFiles = new ArrayList<>();
```

Comment:
Couldn't you make this field final and initialize it in the constructor with an immutable list?